tests/capi/unittest_capi_remote_service.cc

```diff
@@ -199,7 +204,7 @@ TEST_F (MLRemoteService, registerPipeline)
   status = ml_option_set (server_option_h, "connect-type", server_connect_type, g_free);
   EXPECT_EQ (ML_ERROR_NONE, status);
 
-  g_autofree gchar *pipeline_desc = g_strdup ("fakesrc ! fakesink");
+  gchar *pipeline_desc = g_strdup ("fakesrc ! fakesink");
```

Comment:
memory leak?

Reply:
Yes, it might be a memory leak. But ```g_autofree``` might be a dangling pointer.

I am not sure it is ok to use `g_autofree``` pointers as arguments to functions.
We need to define the behavior of the pointer parameters.

Reply:
@jaeyun-jung g_free has been added.